ci-operator/config/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master.yaml

```diff
@@ -204,6 +204,11 @@ tests:
   steps:
     cluster_profile: vsphere
     workflow: openshift-e2e-vsphere-etcd-scaling
+- as: e2e-aws-etcd-recovery
+  optional: true
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws-etcd-recovery
```

Comment:
While we'll definitely need presubmits for the `openshift/cluster-etcd-opearator` repo, I think we'll need to add the presubmits for `openshift/origin` so we can actually test it out on PRs where we update the test. Otherwise I don't think we'll have a way to test the changes to the test before it goes in.

That config will go here I believe:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/origin/openshift-origin-master.yaml

Reply:
(added it nevertheless)

Reply:
Ah sorry I wasn't aware if we could just run it with `/test`. I thought we needed the presubmit here in order to invoke the target for `/test e2e-aws-etcd-recovery`.

But if we can just run it directly then we can do without adding it to the config. It's probably not a good idea to run this by default for all PRs on origin.

Reply:
yep we will be able to run it from the rehearsal and with /test. The presubmits for CEO and etcd should stay I presume? I'll go and remove the origin presubmits then.

Reply:
added the one for etcd 4.14, CEO stays on master for the time being and removed the origin entry.